Hosting

Mingling94 opened the pull request:
Hosting plugin. Not exactly pretty, and the configuration settings have to be rounded because JSON numbers get automatically converted to JS floating point numbers and thus lose precision while gaining decimals.

This may necessitate the API to return numbers as strings, e.g. `"1123123"` instead of `1123123`

seveibar commented:
build failed for api limiting reasons- not b/c it wouldn't actually build (I hope)

lukechampine commented:
FYI: If you have one-off build failures like this, you can `commit --amend` the last commit and `push --force` it, which will restart the build.

Mingling94 committed:
Library folder renamed to Host to start work

Mingling94 committed:
Renamed to Hosting, added barebones CSS

Mingling94 committed:
Styled in the manner of Wallet plugin

Mingling94 committed:
Added tooltip code

Have yet to test it

Also changed more " to '

Mingling94 committed:
Managed incorporating the other PR's effects

Hosting's css pointers were off and there were still lots of double
quotes around.

Mingling94 committed:
Adapted Seve's code to current style

Mingling94 committed:
Adapting hosting.js's old code to new structure

Work in progress

Mingling94 committed:
Lots of fields are now working

Mingling94 committed:
Hosting Configuration Properties Work

Mingling94 committed:
Adjust plugin header font sizes

Mingling94 committed:
Adapted hosting to the new API IPC paradigm

Mingling94 committed:
Stylized Hosting and Wallet to have bigger headers

also generally increased the size of everything. Maybe it's just my
hidpi screen

Mingling94 commented:
@lukechampine This PR should be reviewed soonish so that I can keep updating Hosting with the greater design changes in my other PRs
